src/nccl_ofi_sendrecv.c

```diff
@@ -1937,6 +1940,10 @@ static inline nccl_net_ofi_sendrecv_req_t *prepare_send_req(nccl_net_ofi_sendrec
 {
 	nccl_net_ofi_sendrecv_req_t *req = NULL;
 
+	if (s_comm == NULL) {
```

Comment:
This is in the critical path and should have an OFI_UNLIKELY.

Reply:
Change made.